src/PackageProject.cmake

```diff
@@ -154,7 +154,9 @@ function(package_project)
     LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT shlib
     ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT lib
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT bin
-    PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${_PackageProject_NAME}" COMPONENT dev)
+    PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${_PackageProject_NAME}" COMPONENT dev
+    FILE_SET HEADERS # NOTE: required from CMake to install the FILE_SET HEADERS too!
```

Comment:
It fails on some of the CI configurations.

Reply:
On which version, CMake has started to require this argument?

Reply:
the [HEADER_SET](https://cmake.org/cmake/help/v3.24/prop_tgt/HEADER_SET.html?highlight=file_set) is new in `CMake v3.23`